doc/quick_start_guide.md

```diff
@@ -156,9 +156,16 @@ Of special interest are:
 
 Apache Flume [web site](http://flume.apache.org/) is also another interesting pointer to learn more about the base technlogy used by Cygnus.
 
-##Contact
-Fermín Galán Marquez (fermin.galanmarquez@telefonica.com)
-<br>
-Francisco Romero Bueno (francisco.romerobueno@telefonica.com)
-<br>
-Germán Toro del Valle (german.torodelvalle@telefonica.com)
+##Reporting issues and contact information
```

Comment:
I finally maintained the same section in both documents. I will think about the proposed possibility for the future.

Fixes appearing in https://github.com/telefonicaid/fiware-cygnus/commit/ba27b6cb37e70e5d3b30d25f63b9d5a19bcee92e